endorse/commit.go

```diff
@@ -63,20 +63,13 @@ type VersionControl interface {
 	GetChangeOps(ctx context.Context) (ChangeOps, error)
 	// RetriableError returns true if TryCommit's provided error is retriable.
 	RetriableError(err error) bool
-	// Result returns a successful commit's representation given a successful TryCommit's result and
+	// Result stores a successful commit's representation given a successful TryCommit's result and
```

Comment:
Stores where?

Reply:
In the VCS object itself. I really only need this for prod_test where I confirm the certPath.